docs/current_docs/features/snippets/debugging-1/java/src/main/java/io/dagger/modules/mymodule/MyModule.java

```diff
@@ -11,7 +11,7 @@
 @Object
 public class MyModule {
   @Function
-  public String foo() throws ExecutionException, DaggerQueryException, InterruptedException {
+  public String foo() throws ExecutionException, DaggeQueryException, InterruptedException {
```

Comment:
```suggestion
  public String foo() throws ExecutionException, DaggerQueryException, InterruptedException {
```

Just a small typo here

Reply:
yes sorry but the formatter is crazy not manage this situation :(